wp-content/themes/goonj-crm/assets/styles/main.scss

```diff
@@ -293,6 +294,31 @@ form input {
 .font-sans {
     font-family: Open Sans !important;
 }
+
+@media (max-width: 768px) {
+    .login-submit {
+        display: flex;
+        justify-content: center;
+    }
+    .button-primary {
+        width: 100%;
+        max-width: 300px;
+    }
+    .mobile-margins-l24-r30 {
+        margin-left: 24px !important;
+        margin-right: 30px !important;
+        margin-bottom: 30px !important;
+    }
+    .mobile-margin-left-30 {
+        margin-left: 30px !important;
+    }
+
+    .mobile-margins-l30-r30 {
```

Comment:
what does this mean? 

From what I can read, it is for mobile screens, but you are putting it inside the 768px breakpoints.

Also, we don't have to specify margins class for each size indipendently, I believe we have $spacers and $sizes defined in `variable.scss` which can be used directly. 

Checkout this, which can help you to get more details on how these things works - https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
@belwalshubham I read it `min-width`.

Actually you don't have to specify`max-width`, instead, override it in md.scss file

Reply:
fixed